flame/httpssession.cpp

```diff
@@ -118,9 +123,14 @@ int on_frame_recv_callback(nghttp2_session *session,
 {
     auto class_session = static_cast<HTTPSSession *>(user_data);
     switch (frame->hd.type) {
-    case NGHTTP2_SETTINGS:
-        class_session->settings_received();
-        break;
+        case NGHTTP2_SETTINGS:
+            class_session->settings_received();
+            break;
+        case NGHTTP2_DATA:
+            if (frame->hd.flags & NGHTTP2_FLAG_END_STREAM) {
+                auto data = class_session->_recv_chunks[frame->data.hd.stream_id];
+                class_session->process_receive(data.data(), data.size());
```

Comment:
Will this free the resources from the vector here? I'm trying to understand if this will consume but not release memory.